ereolen_support/templates/ereolen-support--help-text.tpl.php

```diff
@@ -1,17 +1,17 @@
 <?php
+
 /**
  * @file
  * Template for showing a help message to the user.
  */
 ?>
 <div class="form-item">
   <?php foreach ($problem_values as $problem) : ?>
-    <?php $text = variable_get('ereolen_support_problem_help_' . $problem->tid)['value'] ?>
+    <?php $text = variable_get('ereolen_support_problem_help_' . $problem->tid)['value'] ?? NULL ?>
```

Comment:
Should some of these ` ?? NULL` in the PR have been ` ?? '',` the empty string?

Reply:
No, we wan't to know that no (variable) value is set.